holoviews/ipython/__init__.py

```diff
@@ -44,6 +44,7 @@ def setUp(self):
         except Exception:
                 raise SkipTest("IPython could not be started")
 
+        self.ip.displayhook.flush = lambda: None  # To avoid gc.collect called in it
```

Comment:
This is not directly related to any failing test but gives a significant overhead when running the test suite.